gorb/gorb.go

```diff
@@ -0,0 +1,323 @@
+/*
+Package gorb provide the ability to register and deregister the backend
+*/
+package gorb
+
+import (
+	"errors"
+
+	"time"
+
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"io"
+
+	"path"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/hashicorp/go-multierror"
+	"github.com/sky-uk/feed/controller"
+)
+
+// HealthcheckArgs defines health check URI
+type HealthcheckArgs struct {
+	Method string `json:"method"`
+	Path   string `json:"path"`
+	Expect string `json:"expect"`
+}
+
+// Pulse defines backend health check
+type Pulse struct {
+	TypeHealthcheck string          `json:"type"`
+	Args            HealthcheckArgs `json:"args"`
+	Interval        string          `json:"interval"`
+}
```

Comment:
Wait this is confusing. Either call it "Healthcheck" and "HealthcheckArgs" or "Pulse" and "PulseArgs". I'm okay with "Pulse" since that is the gorb naming. @alangibson01 wdyt?

Reply:
As long as it's consistent, I don't see it makes too much of a difference. Personally, even though pulse is the name used for health checking in GORB, it feels ambiguous to me. But I'm not going to kick up a fuss over it if nobody else minds :)

Reply:
ok, pulse it is